packages/foam-vscode/src/services/variable-resolver.ts

```diff
@@ -251,19 +262,22 @@ export class Resolver implements VariableResolver {
     const result = this.promises.get(name);
     return result;
   }
-}
 
-async function resolveFoamTitle() {
-  const title = await window.showInputBox({
-    prompt: `Enter a title for the new note`,
-    value: 'Title of my New Note',
-    validateInput: value =>
-      value.trim().length === 0 ? 'Please enter a title' : undefined,
-  });
-  if (title === undefined) {
-    throw new UserCancelledOperation();
+  async resolveFoamTitle() {
```

Comment:
Should we be able to use the old version of this function at this point?

Reply:
Won't that require prompting the user twice if the value isn't stored somewhere?

Reply:
because of the check in line 129 (`} else if (!this.promises.has(name)) {`) I don't believe that would be the case